server.js

```diff
@@ -7517,6 +7517,70 @@ function(data, match, end, ask) {
   }
 });
 
+// Distelli integration
+camp.route(/^\/distelli\/([^/]+)\/([^/]+)(?:\/(.+))?\.(svg|png|gif|jpg|json)$/, cache({
+  queryParams: ['token'],
+  handler: (data, match, sendBadge, request) => {
+    const type = match[1]; // build or deploy
```

Comment:
Could you modify the regex so it accepts only these values?

Reply:
Sure 👍